### PR TITLE
Few more unused use statement in config directory

### PR DESCRIPTION
**Pull request by narendravaghela**
It simply removes on unused use statement in 
- config/bootstrap.php
- config/bootstrap_cli.php

### CODE REVIEW

File: config/bootstrap.php [hunk 1]
```diff
@@ -41,7 +40,6 @@
 use Cake\Http\ServerRequest;
 use Cake\Log\Log;
 use Cake\Mailer\Email;
-use Cake\Utility\Inflector;
```
Comment (saeideng) [inline config/bootstrap.php]: for these lines 
https://github.com/cakephp/app/blob/aeca5e2a4c69999beaa743cd41286407fe3b37e3/config/bootstrap.php#L200-L203
we can not remove it

Comment (narendravaghela) [reply config/bootstrap.php]: Oh yea, thank you for the notice.
Actually, that code was commented, so let me add that back.

Comment (narendravaghela) [reply config/bootstrap.php]: Added the statement back to file.

**Comment by saeideng** [comment]
Thank you :+1:

**Pull request merged**